File: Server/runes/SetHealth.js
var entities = require('../entityManager');


exports.execute = function(rune, state) {
        var card = entities.getEntity(rune["cardGuid"], state);
        var amount = rune["amount"];
        card["totalHealth"] = amount;
        if(card["currentHealth"] > card["totalHealth"])
        {
            card["currentHealth"] = card["totalHealth"];
        }
}

exports.CreateRune = function(cardGuid, amount)
{
    var rune = {
        "runeType":"SetHealth",
        "cardGuid":cardGuid,
        "amount":amount
    }
    
    return rune;
}

exports.canSee = function (params) {
    return true;
}

File: Assets/Scripts/Runes/SetHealth.cs
using UnityEngine;
using System.Collections;

public class SetHealth : Rune {

    public string card_uid { get; set; }
    public int amount { get; set; }

    public override void Execute(System.Action action)
    {
		var ent = EntityManager.Singelton.GetEntity(card_uid) as damageable;
        ent.SetHealth(amount);
        action();
    }

    public override void OnGUI()
    {

    }
}
